pkg/brevity/group.go

```diff
@@ -46,6 +46,10 @@ type Group interface {
 	Fast() bool
 	// VeryFast is true is the group's speed is above 900kts ground speed or 1.5 Mach.
 	VeryFast() bool
+	// MergedWith is the number of friendlies this group is merged with.
```

Comment:
ignoring this comment until callsign registry is implemented I think is fine.

Reply:
> you could use checking in as a way to add people and their callsigns. "skyeye, marlan 69, checking in as fragged, request alpha check " -> match unit with name closest to "marlan 69" then register that unit as "marlan 69".

No need for the check-in, GCI can already do it via a combination of the Name label on the trackfile, ParsePilotCallsign() and IsClientOnFrequency().

However I think we need to revisit this later because there's a data problem:

1. We can't directly distinguish an AI aircraft from a player/client aircraft using ACMI from the official TacView exporter
2. We don't have access to the mission editor callsigns like Dodge 11 for AI aircraft, instead we get whatever the mission editor put in the unit name field which is an arbitrary string

We can potentially get that data by doing this https://github.com/dharmab/skyeye/issues/65#issuecomment-2323003488, at which point we could come back and change this